docs/API.md

```diff
@@ -432,6 +432,7 @@ Init only:
 Dynamic:
 
 - **lngLat**: [`LngLatLike`](https://maplibre.org/maplibre-gl-js/docs/API/types/maplibregl.LngLatLike/)
+- **rotation**: `number`
```

Comment:
I wonder if this can be auto generated using typedoc or something...

Reply:
in some other projects of mine, I am using [compodoc](https://compodoc.app/) to generate documentation from code which works fairly well. however, the output is html and not markdown.

Reply:
Npm trends show that typedoc is more popular, it can also generate markdown using a plugin, but I thought a simple iframe would do the trick with some static generated html files.
But there will be a need to update the code comments.
This is out of scope for this PR obviously.
I've started looking at this, typedoc generates a decent html for the API. If you want to pursue this, let me know.

Reply:
I've created the following issue for this:
- #150